### DISCUSSION TITLE
School Has Restricted Emails

### DISCUSSION THREAD

**Discussion by pati08** [community/community#107751] [GitHub Education]
### Hi everyone,

I'm trying to begin the process of getting approved. Unfortunately, my school's entry requires me to verify my school email address which is restricted to receiving emails only from specific senders. None of these senders are GitHub. The Technical Director agreed to add it, but needs a specific email to add. What email will it come from?

**Comment by mw-xyz** [comment]
Should be `education@github.com` but ask if all emails from domain GitHub.com could be unblocked. Let me know if this helps
